hax/hax/bytecount.py

```diff
@@ -147,11 +150,13 @@ def _execute(self, motr: Motr):
                     pver_bc = self._calculate_bc_per_pver(pver_items)
                     self.consul.update_bc_for_dg_category(pver_bc, pver_items)
                 except HAConsistencyException:
```

Comment:
Both the exceptions can be caught in a single except clause.
```
except (HAConsistencyException, BytecountException) as e
```

Reply:
As per request, merging this PR and handle the suggested change in separate ticket. cc @vaibhavparatwar, @Shreya-18 .